iphone/Maps/UI/Help/AboutController/AboutController.swift

```diff
@@ -492,14 +492,13 @@ private extension AboutController {
       return
     }
     // From iOS 14, it is possible to change the default mail app, and mailto should open a default mail app.
-    if MWMMailViewController.canSendMail() {
-      let vc = MWMMailViewController()
-      vc.mailComposeDelegate = self
-      vc.setSubject(subject)
-      vc.setMessageBody(body, isHTML:false)
-      vc.setToRecipients(toRecipients)
-      vc.navigationBar.tintColor = UIColor.whitePrimaryText()
-      self.present(vc, animated: true, completion:nil)
```

Comment:
Maybe dont' remove MWMMailViewController?
Lets make as we discussed earlier in the issue:
if (canOpenDefaultApp) {
 open default...
} else MWMMailViewController.canSendMail() {
  send with internal mail controller
} else {
  show error
}

Reply:
So if I'm not mistaken it will never reach the first else and execute the MWMMailViewController.canSendMail() because it will always open with the default email app, even if that is the native app. If it can't open with URL then it shouldn't be able to open with .canSendMail() and would go to the error